app/assets/javascripts/cascade/query-string.js

```diff
@@ -1,67 +1,71 @@
 $(function () {
```

Comment:
The only thing I did in this whole file was add a conditional to check if the current page was using omninav v2